app/src/Controllers/HomeController.php

```diff
@@ -22,8 +23,14 @@ public function __construct($view, LoggerInterface $logger, $user)
     public function displayHomepage(Request $request, Response $response, $args)
     {
         $this->logger->info("Home page action dispatched");
-		
-        $this->view->render($response, 'homepage.twig');
+        
+        if($user = Sentinel::forceCheck()){
```

Comment:
Couldn't it be more convenient to call Sentinel::forceCheck static method directly in the array sent to the twig file?